docs/manual/usage/configuration.md

```diff
@@ -0,0 +1,112 @@
+# Configuration Example {#sec-usage-configuration}
+
+A fresh install of Home Manager will generate a minimal
+`~/.config/home-manager/home.nix` file containing something like
+
+``` nix
+{ config, pkgs, ... }:
+
+{
+  # Home Manager needs a bit of information about you and the
+  # paths it should manage.
+  home.username = "jdoe";
+  home.homeDirectory = "/home/jdoe";
+
+  # This value determines the Home Manager release that your
+  # configuration is compatible with. This helps avoid breakage
+  # when a new Home Manager release introduces backwards
+  # incompatible changes.
+  #
+  # You can update Home Manager without changing this value. See
+  # the Home Manager release notes for a list of state version
+  # changes in each release.
+  home.stateVersion = "23.11";
+
+  # Let Home Manager install and manage itself.
+  programs.home-manager.enable = true;
+}
+```
+
+You can use this as a base for your further configurations.
+
+::: note
+If you are not very familiar with the Nix language and NixOS modules
+then it is encouraged to start with small and simple changes. As you
+learn you can gradually grow the configuration with confidence.
+:::
```

Comment:
Doesn't seem to render correctly:
![swappy-20231129_230226](https://github.com/nix-community/home-manager/assets/798147/a2be99e1-3f24-4fc1-8edd-a86c475da47c)

Reply:
The other `::: note` elements are rendered similarly…